persistence/provider/memory/eventstore.go

```diff
@@ -98,7 +137,6 @@ func (r *eventStoreResult) Next(
 
 // Close closes the cursor.
 func (r *eventStoreResult) Close() error {
-	r.query.MinOffset = math.MaxUint64
```

Comment:
You the equivalent line in the BoltDB implementation, any reason not to keep this one? Or otherwise, any reason not to get rid of both of them?

They're not strictly necessary, I just added them as a bit of protection to make sure you couldn't read anything after calling `Close()`, but I guess this one never really did anything.

Reply:
I am happy with either leaving or getting rid of them both.

Reply:
Ok, if the tests pass without them, just get rid of them. I am happy to leave the behavior of a closed cursor's `Next()` method as undefined.

I'm working on the theory that if it had seemed important at the time I would have added a test for it.

Reply:
Haha 😄 , just added that one back before I saw your message. Let me take them both out and re-run the tests.

Reply:
Ok, local tests passed.